### PR TITLE
snappy: use modern udev/apparmor profile names

### PR DESCRIPTION
**Pull request by zyga**
This patch switches udev and apparmor profile names to be identical,
"snap.$SNAP.$APP" and in line with what is done by the interfaces system.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

**Comment by mvo5** [comment]
:+1:

File: snappy/binaries.go [hunk 1]
```diff
@@ -114,10 +113,10 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
 		SnapArch:    arch.UbuntuArchitecture(),
 		SnapPath:    pkgPath,
 		Version:     app.Snap.Version,
-		UdevAppName: fmt.Sprintf("%s.%s", app.Snap.Name(), app.Name),
+		UdevAppName: fmt.Sprintf("snap.%s.%s", app.Snap.Name(), app.Name),
```
Comment (pedronis) [inline snappy/binaries.go]: there should be a helper in snap/ or on Info to make this,  having this kind of logic in many places that need to agree is what made such a pain to switch things around to start with

File: snappy/binaries.go [hunk 2]
```diff
@@ -114,10 +113,10 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
...
 		Home:        "$HOME",
 		Target:      actualBinPath,
-		AaProfile:   aaProfile,
+		AaProfile:   fmt.Sprintf("snap.%s.%s", app.Snap.Name(), app.Name),
```
Comment (pedronis) [inline snappy/binaries.go]: same

File: snappy/binaries.go (hunk 1 above)
Comment (zyga) [inline snappy/binaries.go]: Replaced with ``app.SecurityTag()`` now, thanks for pointing this out.

File: snappy/binaries.go (hunk 2 above)
Comment (zyga) [inline snappy/binaries.go]: Ditto

**Comment by pedronis** [comment]
+1

**Pull request merged**